src/content/en/fundamentals/engage-and-retain/push-notifications/how-push-works.md

```diff
@@ -0,0 +1,160 @@
+
+A `PushSubscription` contains all the information we need to send a push message to that user.
+You can "kind of" think of this as a ID for that user's device.
+
+This is all done in JavaScript with the [Push
+API](https://developer.mozilla.org/en-US/docs/Web/API/Push_API).
+
+Before subscribing a user you'll need to generate a set of
+"application server keys", which we'll cover later on.
+
+The application server keys, also known as VAPID keys, are unique to your server. They allow a
+push service to know which application server subscribed a user and ensure that it's the same
+server triggering the push messages to that user.
+
+Once you've subscribed the user and have a `PushSubscription`, you'll need to send the
+`PushSubscription` details to your backend / server.  On your server, you'll save this
+subscription to a database and use it to send a push message to that user.
+
+![Make sure you send the PushSubscription to your backend.](./images/svgs/browser-to-server.svg)
+
+## Step 2: Send a Push Message
+
+When you want to send a push message to your users you need to make an API call to a push
+service. This API call would include what data to send, who to send the message to and any
+criteria about how to send the message.
+
+Some questions you might be asking yourself:
+
+- Who and what is the push service?
+
+- What does the API look like? Is it JSON, XML, something else?
+
+- What can the API do?
+
+### Who and What is the Push Service?
+
+A push service has the job of receiving requests and delivering push messages if the request is
+valid.
+
+Each browser can use any push service they want, it's something developers have no control
+over. This isn't a problem because every push service expects the **same** API call. Meaning
+you don't have to care who the push service is, you just need to make sure that your API call
+is valid.
+
+To get the appropriate URL to trigger a push message (i.e. the URL for the push service) you
+just need to look at the `endpoint` value in a `PushSubscription`.
+
+Below is an example of the values you'll get from a **PushSubscription**:
+
+	{
+	  "endpoint": "https://random-push-service.com/some-kind-of-unique-id-1234/v2/",
+	  "keys": {
+	    "p256dh" :
+	"BNcRdreALRFXTkOOUHK1EtK2wtaz5Ry4YfYCA_0QTpQtUbVlUls0VJXg7A8u-Ts1XbjhazAkj7I99e8QcYP7DkM=",
+	    "auth"   : "tBHItJI5svbpez7KI4CCXg=="
+	  }
+	}
+
+The **endpoint** in this case is
+*https://random-push-service.com/some-kind-of-unique-id-1234/v2/*. The push service would be
+'random-push-service.com' and each endpoint is unique to a user, indicated with
+'some-kind-of-unique-id-1234'. As you start working with push you'll notice this pattern.
+
+We'll cover what the **keys** value is for later on.
+
+### What does the API look like?
+
+I mentioned that every web push service expects the same API call. That API is the [**Web Push
+Protocol**](https://tools.ietf.org/html/draft-ietf-webpush-protocol).
```

Comment:
The phrase 'Web Push Protocol' sticks out already because it's blue, so you don't need it bolded. Please remove.